webpack.config.js

```diff
@@ -103,13 +103,9 @@ const webpackConfig = {
 					amd: false,
 				},
 			},
-			{
-				test: /\.jsx?$/,
-				loader: 'babel-loader',
-				exclude: /node_modules/,
-			},
 			{
 				test: /\.js?$/,
```

Comment:
Should this be `/\.jsx?$/`? I think it was errant before - unless we wanted to match `.j` files...

Reply:
@jeffstieler Do we need to process `jsx` files, maybe leaving the x out we can discourage naming files jsx? good point about the `?`. How about `/\.js$/` ?

Reply:
I've fixed this.